### DISCUSSION TITLE
Copilot refuses to activate for student account

### DISCUSSION THREAD

**Discussion by LarsVC4** [community/community#31137] [Copilot Conversations] [answered]
Hi, 
I have a problem with github copilot where it doesn't want to activate even though I have an student/PRO account. 
It shows me this error: "GitHub Copilot could not connect to server. Extension activation failed: "Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended". 

Does anyone know how to fix this?

**Comment by melvincwng** [comment, marked as answer]
@LarsVC4, you might have to sign out of your GitHub Account in the IDE which you are using.

If you are using VSCode, there should be a icon shaped like a cog, at the bottom left corner of your screen. Click on it, logout of your account and relogin again.

If you are using other IDEs, do a Google search and check how to logout & then login back to your GitHub account on your IDE.

Once you have relogin, it should probably work.

Otherwise, you might have to contact GitHub Support on this issue. Alternatively, you can wait for other community members to chime in and provide alternate solutions.

Hope this helps you.

**Reply by LarsVC4** [reply]
Thank you so much. It turns out I was still signed in in an old account instead of the student one. So it works now!

**Reply by melvincwng** [reply]
@LarsVC4 Yes that's what I thought as well. Probably needed to relogin to get a new access token (which contains the GitHub Copilot privileges since you are a student/PRO member).

Glad this has helped you, take care and have a good day ahead 😄

**Reply by LarsVC4** [reply]
You to <3!

**Reply by Pzixel** [reply]
Hello, I'm using IDEA CE and have this error. Could you please elaborate which steps should I take since I tried to do some shenanigans but with no success.

Also it says that copilot is free for "popular open source projects" - where can I found list of them? How do I check if I apply to it or not?

**Comment by vanchuong201** [comment]
I'm a student, and I have a verified account:
<img width="649" alt="image" src="https://user-images.githubusercontent.com/7368585/211216627-570bc1d2-fd18-4036-a7a9-4df058d7133f.png">

But I still get the warning: " Thank you for participating in the GitHub Copilot Technical Preview. Your extended access has ended."
<img width="796" alt="image" src="https://user-images.githubusercontent.com/7368585/211216731-a9ffeb3b-4fe0-4122-80cc-fc413829f889.png">
I can't "Enable GitHub Copilot". 
I use Webstorm IDE.
------------
Excuse me. My English is not good !

**Reply by otkrickey** [reply]
I have created a discussion for today's case. Please share your situation. #43634

**Reply by HeJinYang123** [reply]
hello, I have the same problem with you

**Reply by MdWahiduzzamanEmon** [reply]
I am also facing the same issue.

**Reply by otkrickey** [reply]
@MdWahiduzzamanEmon 
Solved: #43634

**Comment by hsh9544** [comment]
try visiting [this link](https://github.com/github-copilot/free_signup). It solved the problem for me.
